cpp/src/Slice/Parser.cpp

```diff
@@ -659,15 +659,15 @@ namespace
 
         // Fix any link tags using the provided link formatter.
         const string link = "{@link ";
-        pos = comment.find(link, pos);
+        pos = comment.find(link);
         while (pos != string::npos)
         {
             string::size_type endpos = comment.find('}', pos);
             if (endpos != string::npos)
             {
                 // Extract the linked to identifier.
                 string::size_type identStart = comment.find_first_not_of(" \t", pos + link.size());
-                string::size_type identEnd = comment.find_last_not_of(" \t", endpos) + 1;
```

Comment:
A bug in my previous comment-formatting PR that caused us to read one character past the link's identifier.